docs/user-guide/vpcs.md

```diff
@@ -181,11 +180,12 @@ spec:
 
 ## VLANNamespace
 
-A VLANNamespace defines non-overlapping IPv4 ranges for VPC subnets. Each VPC belongs to a specific IPv4 namespace.
+A `VLANNamespace` defines a set of VLAN ranges available for attaching servers to switches. Each switch can belong to one or more
+disjoint VLANNamespaces.
 
 ```yaml
 apiVersion: wiring.githedgehog.com/v1alpha2
-kind: VLANNamespace
+kind: VLANNamespac`
```

Comment:
typo

Reply:
Oh boy. I was hoping to fix typos on my way through the documentation.... not meaning to add new ...